<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1227</width>
    <height>1162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>1000</width>
        <height>90</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1">
       <item row="0" column="5">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(85, 87, 83);</string>
         </property>
         <property name="text">
          <string>TOD Port</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(85, 87, 83);</string>
         </property>
         <property name="text">
          <string>TOD ID</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="LineEdit" name="todPortLineEdit"/>
       </item>
       <item row="0" column="12">
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="14">
        <widget class="LineEdit" name="vobcPortLineEdit"/>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="LineEdit" name="todIpLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>142</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>58</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(85, 87, 83);</string>
         </property>
         <property name="text">
          <string>TOD IP</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="9" colspan="3">
        <widget class="LineEdit" name="todIdLineEdit"/>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="15">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <widget class="QRadioButton" name="manualRadioBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Manual</string>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QRadioButton" name="autoRadioBtn">
         <property name="text">
          <string>Automatic</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="5" colspan="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>150</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="8" colspan="2">
        <widget class="LineEdit" name="sendCycleLineEdit">
         <property name="statusTip">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="11" colspan="2">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>39</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="13" colspan="2">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>101</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="setBtn">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(233, 185, 110);
color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="15">
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(85, 87, 83);</string>
         </property>
         <property name="text">
          <string>Communication Mode</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QLabel" name="label_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(85, 87, 83);</string>
         </property>
         <property name="text">
          <string>Send cycle</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="10">
        <widget class="QLabel" name="label_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(85, 87, 83);</string>
         </property>
         <property name="text">
          <string>ms</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>41</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="13">
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(85, 87, 83);</string>
         </property>
         <property name="text">
          <string>VOBC Port</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1000</width>
        <height>520</height>
       </size>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Send Data</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="4,3,0,0">
        <item>
         <widget class="QTableWidget" name="sendTableWidget">
          <property name="statusTip">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="infoTab">
           <attribute name="title">
            <string>Info Id</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="faultTab">
           <attribute name="title">
            <string>Fault  Id</string>
           </attribute>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>TCMS Data:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tcmsTableWidget">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Receive Data</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="receiveTableWidget"/>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>TCMS Data:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="TextEdit" name="recTCMSData_TextEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>1000</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_5" columnstretch="3,4,1" columnminimumwidth="3,4,1">
       <item row="0" column="2">
        <widget class="QPushButton" name="connectBtn">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(233, 185, 110);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>891</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="tipsLabel">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1227</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Open"/>
   </widget>
   <addaction name="menuFile_F"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TextEdit</class>
   <extends>QTextEdit</extends>
   <header>Gui/TextEdit.h</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">Gui/LineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
